pkg/driver/driver_azure.go

```diff
@@ -786,23 +773,6 @@ func (clients *azureDriverClients) waitForDataDiskDetachment(ctx context.Context
 	return nil
 }
 
-func (clients *azureDriverClients) powerOffVM(ctx context.Context, resourceGroupName string, vmName string) error {
```

Comment:
@dkistner Do we plan to use this in near future, or should it better be removed ?

Reply:
We can always re-introduce it if needed.